milli/src/search/criteria/exactness.rs

```diff
@@ -162,11 +162,11 @@ fn resolve_state(
     use State::*;
     match state {
         ExactAttribute(mut allowed_candidates) => {
-            let query_len = query.len() as u32;
+            let query_len = query.len() as u8;
```

Comment:
I am not quite fine with this cast are you sure that we will not have more than 256 words in the query?